src/Test/Perf/Runner/Benchview.cs

```diff
@@ -1,152 +1,164 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-using Roslyn.Test.Performance.Utilities;
-using System.Collections.Generic;
+
+using System;
 using System.IO;
-using System.Xml;
+using System.Linq;
+
+using Roslyn.Test.Performance.Utilities;
 using static Roslyn.Test.Performance.Utilities.TestUtilities;
-using static Roslyn.Test.Performance.Runner.Tools;
 
 namespace Roslyn.Test.Performance.Runner
 {
     public static class Benchview 
     {
-        private const string s_BenchviewPath = @"\\vcbench-srv4\benchview\uploads\vibench";
+        const string sasEnvVar = "BV_UPLOAD_SAS_TOKEN";
```

Comment:
Thanks, I've updated the PR to match the guidelines. Please let me know if you see anything else